dotcom-rendering/src/web/components/ArticleBody.tsx

```diff
@@ -114,6 +116,16 @@ const globalLinkStyles = (palette: Palette) => css`
 const isRecipe = (tags: TagType[]): boolean =>
 	tags.some(({ id }) => id === 'tone/recipes');
 
+// CAPI only supports certain languages. If CAPI doesn't recognise the language,
+// it defaults to `en`. We should filter out `en` so we don't set an incorrect value.
+// See https://github.com/guardian/content-api/blob/main/porter/src/main/scala/com.gu.contentapi.porter/integration/LanguageDetector.scala#L17
+const decideLanguage = (language = ''): string | undefined =>
+	language != 'en' ? language : undefined;
+
+const decideLanguageDirection = (
+	isRightToLeftLang = false,
+): 'rtl' | undefined => (isRightToLeftLang ? 'rtl' : undefined);
```

Comment:
```suggestion
): 'rtl' | undefined => isRightToLeftLang ? 'rtl' : undefined;
```

Reply:
I thought so too! but `prettier` [adds them back in](https://github.com/guardian/dotcom-rendering/actions/runs/4223115519/jobs/7332452396#step:5:10)